src/test/java/gim/logic/commands/EditCommandTest.java

```diff
@@ -1,157 +1,158 @@
-                new EditExerciseDescriptorBuilder().withName(VALID_NAME_BENCH_PRESS).build());
-
-        String expectedMessage = String.format(EditCommand.MESSAGE_EDIT_EXERCISE_SUCCESS, editedExercise);
-
-        Model expectedModel = new ModelManager(new ExerciseTracker(model.getExerciseTracker()), new UserPrefs());
-        expectedModel.setExercise(model.getFilteredExerciseList().get(0), editedExercise);
-
-        assertCommandSuccess(editCommand, model, expectedMessage, expectedModel);
-    }
-
-    @Test
-    public void execute_invalidExerciseIndexUnfilteredList_failure() {
-        Index outOfBoundIndex = Index.fromOneBased(model.getFilteredExerciseList().size() + 1);
-        EditExerciseDescriptor descriptor = new EditExerciseDescriptorBuilder()
-                .withName(VALID_NAME_BENCH_PRESS).build();
-        EditCommand editCommand = new EditCommand(outOfBoundIndex, descriptor);
-
-        assertCommandFailure(editCommand, model, Messages.MESSAGE_INVALID_EXERCISE_DISPLAYED_INDEX);
-    }
-
-    /**
-     * Edit filtered list where index is larger than size of filtered list,
-     * but smaller than size of exercise tracker
-     */
-    @Test
-    public void execute_invalidExerciseIndexFilteredList_failure() {
-        showExerciseAtIndex(model, INDEX_FIRST_EXERCISE);
-        Index outOfBoundIndex = INDEX_SECOND_EXERCISE;
-        // ensures that outOfBoundIndex is still in bounds of exercise tracker list
-        assertTrue(outOfBoundIndex.getZeroBased() < model.getExerciseTracker().getExerciseList().size());
-
-        EditCommand editCommand = new EditCommand(outOfBoundIndex,
-                new EditExerciseDescriptorBuilder().withName(VALID_NAME_BENCH_PRESS).build());
-
-        assertCommandFailure(editCommand, model, Messages.MESSAGE_INVALID_EXERCISE_DISPLAYED_INDEX);
-    }
-
-    @Test
-    public void equals() {
-        final EditCommand standardCommand = new EditCommand(INDEX_FIRST_EXERCISE, DESC_ARM_CURLS);
-
-        // same values -> returns true
-        EditExerciseDescriptor copyDescriptor = new EditExerciseDescriptor(DESC_ARM_CURLS);
-        EditCommand commandWithSameValues = new EditCommand(INDEX_FIRST_EXERCISE, copyDescriptor);
-        assertTrue(standardCommand.equals(commandWithSameValues));
-
-        // same object -> returns true
-        assertTrue(standardCommand.equals(standardCommand));
-
-        // null -> returns false
-        assertFalse(standardCommand.equals(null));
-
-        // different types -> returns false
-        assertFalse(standardCommand.equals(new ClearCommand()));
-
-        // different index -> returns false
-        assertFalse(standardCommand.equals(new EditCommand(INDEX_SECOND_EXERCISE, DESC_ARM_CURLS)));
-
-        // different descriptor -> returns false
-        assertFalse(standardCommand.equals(new EditCommand(INDEX_FIRST_EXERCISE, DESC_BENCH_PRESS)));
-    }
-
-}
+//package gim.logic.commands;
```

Comment:
If edit command is causing issue, might want to remove it before merging, we will see after next meeting.